### PR TITLE
[Tutorial Fix] Fix multimodal index page, fix the Github README, AutoMMPredictor --> MultiModalPredictor

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*

*Description of changes:*

Fix multimodal index page and fix the Github README.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1913-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1913/1/index.html

**Comment by szha** [comment]
Job PR-1913-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1913/2/index.html

**Comment by szha** [comment]
Job PR-1913-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1913/3/index.html

File: multimodal/src/autogluon/multimodal/predictor.py [hunk 1]
```diff
@@ -1854,3 +1855,9 @@ def positive_class(self):
             return None
         else:
             return self.class_labels[1]
+
+
+class AutoMMPredictor(MultiModalPredictor):
+    def __init__(self, **kwargs):
+        warnings.warn('AutoMMPredictor has been renamed as "MultiModalPredictor". Consider to use the ')
```
Comment (Innixma) [inline multimodal/src/autogluon/multimodal/predictor.py:1862]: Better wording:

```
AutoMMPredictor has been renamed to MultiModalPredictor. Consider to using MultiModalPredictor instead. Using AutoMMPredictor will raise an exception starting in v0.7.
```

**Review by Innixma — APPROVED** [review]
LGTM! Had one comment

**Review by bryanyzhu — APPROVED** [review]
LGTM

**Review by zhiqiangdon — COMMENTED** [review]
Do we need to test backward compatibility? 

```
from autogluon.multimodal import MultiModalPredictor
predictor = MultiModalPredictor.load("path/to/checkpoint/saved/by/using/AutoMMPredictor")
predictor.fit()
verify_save_and_load()
```
If not backward compatible, users would no longer be able to load the AutoMMPredictor checkpoints after v0.7.

**Comment by Innixma** [comment]
> Do we need to test backward compatibility?
> 
> ```
> from autogluon.multimodal import MultiModalPredictor
> predictor = MultiModalPredictor.load("path/to/checkpoint/saved/by/using/AutoMMPredictor")
> predictor.fit()
> verify_save_and_load()
> ```
> 
> If not backward compatible, users would no longer be able to load the AutoMMPredictor checkpoints after v0.7.

We could make AutoMMPredictor a function that simply returns a MultiModalPredictor + logs the warning, so it isn't its own class anymore. That would avoid breaking in v0.7

File: multimodal/tests/unittests/test_backward_compatibility.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+from autogluon.multimodal import MultiModalPredictor, AutoMMPredictor
+from datasets import AmazonReviewSentimentCrossLingualDataset
+from utils import (
+    download,
+    protected_zip_extraction,
+    get_home_dir,
+)
+
+
+def verify_predictor_save_load(predictor, df, verify_embedding=True, cls=MultiModalPredictor):
```
Comment (zhiqiangdon) [inline multimodal/tests/unittests/test_backward_compatibility.py:17]: I guess we can just do this:

`from test_predictor import verify_predictor_save_load`

**Comment by szha** [comment]
Job PR-1913-12 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1913/12/index.html

**Review by zhiqiangdon — APPROVED** [review]
LGTM. Thanks!

**Comment by szha** [comment]
Job PR-1913-13 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1913/13/index.html

**Pull request merged**